africhi.csl

```diff
@@ -0,0 +1,165 @@
+<?xml version="1.0" encoding="utf-8"?>
+<style xmlns="http://purl.org/net/xbiblio/csl" class="in-text" version="1.0" demote-non-dropping-particle="sort-only" default-locale="en-US">
+<!-- This style was edited with the Visual CSL Editor (http://editor.citationstyles.org/visualEditor/) -->
+  <info>
+    <title>africhi</title>
+    <id>http://www.zotero.org/styles/africhi</id>
+    <link href="http://www.zotero.org/styles/africhi" rel="self"/>
+    <link href="http://www.acm.org/sigs/publications/proceedings-templates" rel="documentation"/>
+    <author>
+      <name>Naeem Esfahani</name>
+      <email>nesfaha2@gmu.edu</email>
+      <uri>http://mason.gmu.edu/~nesfaha2/</uri>
+    </author>
```

Comment:
Per https://github.com/citation-style-language/styles/wiki/Style-Requirements#8---template-link, we prefer it if you delete any authors/contributors from the original style, and add a "template" link instead.